tox.ini

```diff
@@ -1,33 +1,31 @@
 [tox]
 minversion = 3.8.6
 envlist =
-    lint
+    lint,
+    sanity,
+    functional
 skipdist = True
 skip_missing_interpreters = True
 
 [travis]
 python =
-  3.7: lint, functional
+  3.7: lint, sanity, functional
```

Comment:
Let's remove functional until we figure out how to properly test this one

Reply:
`[testenv:functional]` has been completely removed.